src/sdk/accountspace_fs.js

```diff
@@ -31,11 +31,14 @@ const account_id_cache = new LRUCache({
     // TODO: Decide on a time that we want to invalidate
     expiry_ms: Number(config.ACCOUNTS_ID_CACHE_EXPIRY),
     /**
-     * @param {{ _id: string, config_fs: import('./config_fs').ConfigFS }} params
+     * @param {{ _id: string, 
+     * show_secrets: boolean,
+     * decrypt_secret_key: boolean, 
+     * config_fs: import('./config_fs').ConfigFS }} params
      */
     make_key: ({ _id }) => _id,
-    load: async ({ _id, config_fs }) => config_fs.get_identity_by_id(_id, CONFIG_TYPES.ACCOUNT,
-        { show_secrets: true, decrypt_secret_key: true }),
+    load: async ({ _id, show_secrets, decrypt_secret_key, config_fs}) => config_fs.get_identity_by_id(_id, CONFIG_TYPES.ACCOUNT,
```

Comment:
this looks dangerous IMO, we don't want to have an account cache that has in it missing/encryped secret key, this could lead to runtime bugs
this cache code is being used by manage_nsfs and the endpoint, I believe people won't use it carefully